src/main/java/com/uid2/shared/secure/azurecc/MaaTokenPayload.java

```diff
@@ -7,8 +7,14 @@
 @Builder(toBuilder = true)
 public class MaaTokenPayload {
     public static final String SEV_SNP_VM_TYPE = "sevsnpvm";
+    public static final String AZURE_CC_PROTOCOL = "azure-cc";
+    public static final String AZURE_CC_AKS_PROTOCOL = "azure-cc-aks";
+    // the `x-ms-compliance-status` value for ACI CC
```

Comment:
you called it ACI here - should ACI be used in variable names elsewhere e.g. AZURE_CC_ACI_PROTOCOL?